portal/src/OnboardingSurveyScreen.module.css

```diff
@@ -44,7 +44,7 @@
 }
 
 .CompoundChoiceButton {
-  max-width: 368px;
+  @apply max-w-[368px] mobile:max-w-[410px];
```

Comment:
In mobile, it should occupy full width instead of having a max of 410px?

Reply:
Added a commit to fix this.